src/theme/Checkbox.ts

```diff
@@ -3,21 +3,36 @@ import { PartsStyleFunction } from "@chakra-ui/theme-tools"
 
 const baseStyle: PartsStyleFunction<typeof checkboxAnatomy> = () => {
   return {
+    label: {
+      display: "flex",
+      alignItems: "center",
+      ml: 3,
+      fontWeight: "medium",
+    },
     control: {
+      outline: "3px solid",
+      outlineColor: "transparent",
+      p: 2,
+      borderRadius: "md",
       _checked: {
-        borderColor: "brand.500",
+        borderColor: "hsl(182, 100%, 70%)",
         _active: {
-          borderColor: "brand.500",
+          borderColor: "hsl(182, 100%, 70%)",
         },
-        backgroundColor: "brand.500",
+        bg: "hsl(182, 100%, 70%)",
         _hover: {
-          backgroundColor: "brand.500",
-          borderColor: "brand.500",
+          bg: "hsl(182, 100%, 25%)",
+          borderColor: "hsl(182, 100%, 25%)",
         },
       },
+      _focus: {
+        boxShadow: "0 0 0 3px hsla(182, 100%, 70%, 20%)",
+      },
     },
     icon: {
-      color: "white",
+      w: 3,
+      h: 3,
+      color: "#121212",
```

Comment:
Btw, is there any specific reason you sometimes use `hsl` and sometimes `#<color_name>`? I think it would be better to stick to one syntax.

Reply:
Definitely. Sadly we don't have the color palette (yet? 🤔) so I wasn't really strict with the color values.

There are a lot of shades of gray in designs, it's difficult to keep up the pace without the color palette XD
Personally I'd like to stick to HSL(A) color representation model. In my opinion it's much more human readable than hexadecimal representation.

If you agree just please let me know if you spot any occurance of hexadecimal color definition while reviewing my code so I can transform it into HSL.

Reply:
I don't mind. We can use `hsl` although I think that we would have to convert in hex color provided by designers to a HSL, which is an additional step to do 😅 Unless we can convert it straight in Figma?